### PR TITLE
Removes superfluous apostrophe in warning

### PR DESCRIPTION
**Pull request by buley**
Small tweak - noticed my error message showed a grammatical mistake.

### CODE REVIEW

**Comment by MatrixFrog** [comment]
`''` prints as `'` for the formatter that we're using.

**Comment by buley** [comment]
Cool - perhaps I'm seeing then a formatting bug with [closure-compiler-js](https://github.com/google/closure-compiler-js/)

**Comment by tanvirsobhanwe** [comment]
what??

File: src/com/google/javascript/jscomp/CheckSuspiciousCode.java [hunk 1]
```diff
@@ -38,7 +38,7 @@
 
   static final DiagnosticType SUSPICIOUS_SEMICOLON = DiagnosticType.warning(
       "JSC_SUSPICIOUS_SEMICOLON",
-      "If this if/for/while really shouldn''t have a body, use '{}'");
+      "If this if/for/while really shouldn't have a body, use '{}'");
```
Comment (tanvirsobhanwe) [inline src/com/google/javascript/jscomp/CheckSuspiciousCode.java]: now

**Comment by MatrixFrog** [comment]
Ah yes, there may be an inaccurate gwt implementation of the formatter. Good to know.

**Pull request closed without merging**